test/components/DayPickerRangeController_spec.jsx

```diff
@@ -3693,6 +3693,74 @@ describe('DayPickerRangeController', () => {
     });
   });
 
+  describe.only('#onMonchChange', () => {
+    it('sets disableNext as false when maxDate is in visible month and a past month is selected', () => {
+      const wrapper = shallow((
+        <DayPickerRangeController
+          onDatesChange={sinon.stub()}
+          onFocusChange={sinon.stub()}
+          maxDate={today}
+        />
+      ));
+      wrapper.setState({
+        currentMonth: today,
+      });
+      expect(wrapper.state().disableNext).to.equal(true);
+      wrapper.instance().onMonthChange(today.clone().subtract(1, 'month'));
+      expect(wrapper.state().disableNext).to.equal(false);
+    });
+
+    it('sets disablePrev as false when minDate is in visible month and a future month is selected', () => {
+      const wrapper = shallow((
+        <DayPickerRangeController
+          onDatesChange={sinon.stub()}
+          onFocusChange={sinon.stub()}
+          minDate={today}
+        />
+      ));
+      wrapper.setState({
+        currentMonth: today,
+      });
+      expect(wrapper.state().disablePrev).to.equal(true);
+      wrapper.instance().onMonthChange(today.clone().add(1, 'month'));
+      expect(wrapper.state().disablePrev).to.equal(false);
+    });
+  });
+
+  describe.only('#onYearChange', () => {
```

Comment:
```suggestion
  describe('#onYearChange', () => {
```
this would skip the entire rest of the test suite